syz-manager/manager.go

```diff
@@ -544,6 +552,13 @@ func (mgr *Manager) runInstance(index int) (*Crash, error) {
 		return nil, fmt.Errorf("failed to setup port forwarding: %v", err)
 	}
 
+	if mgr.kcovFilter.enableFilter {
+		_, err = inst.Copy(mgr.kcovFilter.pcsBitmapPath)
```

Comment:
I still did not fully understand the problem that mentioned [here](https://github.com/google/syzkaller/pull/2017#discussion_r467928235).